src/system.cpp

```diff
@@ -1358,30 +1358,29 @@ bool System::get_value_info(JsonObject root, const char * command) {
 
 // export status information including the device information
 // http://ems-esp/api/system/info
-// TODO camelCase - #1860
 bool System::command_info(const char * value, const int8_t id, JsonObject output) {
     JsonObject node;
 
     // System
-    node                     = output["System"].to<JsonObject>();
-    node["version"]          = EMSESP_APP_VERSION;
-    node["uptime"]           = uuid::log::format_timestamp_ms(uuid::get_uptime_ms(), 3);
-    node["uptime (seconds)"] = uuid::get_uptime_sec();
+    node              = output["system"].to<JsonObject>();
+    node["version"]   = EMSESP_APP_VERSION;
+    node["uptime"]    = uuid::log::format_timestamp_ms(uuid::get_uptime_ms(), 3);
+    node["uptimeSec"] = uuid::get_uptime_sec();
 #ifndef EMSESP_STANDALONE
     node["platform"]  = EMSESP_PLATFORM;
     node["arduino"]   = ARDUINO_VERSION;
     node["sdk"]       = ESP.getSdkVersion();
-    node["free mem"]  = getHeapMem();
-    node["max alloc"] = getMaxAllocMem();
-    node["free caps"] = heap_caps_get_free_size(MALLOC_CAP_8BIT) / 1024; // includes heap and psram
-    node["used app"]  = EMSESP::system_.appUsed();                       // kilobytes
-    node["free app"]  = EMSESP::system_.appFree();                       // kilobytes
+    node["freMem"]    = getHeapMem();
```

Comment:
'e'

Reply:
Your attention to detail is astonishing!